client.go

```diff
@@ -93,13 +94,16 @@ type requestInput struct {
 	Headers     map[string]string
 	QueryParams map[string]string
 	Body        interface{}
+	RawBody 	[]byte
 	ContentType string
+	Read		bool
```

Comment:
I think `Read` is a confusing name(is a verb), maybe we should call it `ShouldReadBody`